### PR TITLE
`ci`: try `1.8.0` stable release

### PR DESCRIPTION
**Pull request by t-bltg**
- try `1.8` on all OS;
- keep `1.7` on `ubuntu-latest` only to spare credits;
- ~~try to fix ci by bumping `PlotlyBase` to `0.8`~~;
- add [invalidations action check](https://discourse.julialang.org/t/potential-performance-regressions-in-julia-1-8-for-special-un-precompiled-type-dispatches-and-how-to-fix-them/86359/11);
- fix missing `fgcolor` introduced by https://github.com/JuliaPlots/Plots.jl/pull/2621;
- bump `Unzip`.

### CODE REVIEW

**Review by BeastyBlacksmith — APPROVED** [review]

File: Project.toml [hunk 1]
```diff
@@ -51,7 +51,7 @@ NaNMath = "0.3, 1"
 PGFPlotsX = "1"
 PlotThemes = "2, 3"
 PlotUtils = "1"
-PlotlyBase = "0.7"
+PlotlyBase = "0.7 - 0.8"
```
Comment (t-bltg) [inline Project.toml:54]: @BeastyBlacksmith, is that safe ?

Xref https://github.com/sglyon/PlotlyBase.jl/pull/100.

**Comment by BeastyBlacksmith** [comment]
No, that won't work. We are currently working on outsourcing the savefig functionality from PlotlyJS, xref: https://github.com/JuliaPlots/PlotlyJS.jl/issues/435

**Comment by t-bltg** [comment]
Thanks, reverted to `0.7`.

**Comment by t-bltg** [comment]
This took an awful amount of effort to fix :sleepy: .

**Pull request merged**